File: insee-promotion-admin-client/app/components/enum/MatchStatus.js
export class MatchStatus {

    constructor(id, name) {
        this.id = id;
        this.name = name;
    }

    static findById(id) {
        switch(id) {
            case 1 : return INIT;
            case 2 : return PROCESSING;
            case 3 : return DONE;
        }
    }

    static getList() {
        return [INIT, PROCESSING, DONE]
    }
}

export const INIT = new MatchStatus(1, 'Sắp bắt đầu');
export const PROCESSING = new MatchStatus(2, 'Đang diễn ra')
export const DONE = new MatchStatus(3, 'Đã kết thúc');

File: ZaloBotServer/src/main/java/com/vn/insee/zalobot/common/status/MatchStatus.java
package com.vn.insee.zalobot.common.status;

public enum MatchStatus {
    INIT(1), PROCESSING(2), DONE(3);
    private int value;

    MatchStatus(int value) {
        this.value = value;
    }

    public int getValue() {
        return value;
    }

    public void setValue(int value) {
        this.value = value;
    }
}
